packages/react/src/components/ImageToolbar/mantine/ImageToolbar.tsx

```diff
@@ -5,169 +5,100 @@ import {
   InlineContentSchema,
   StyleSchema,
 } from "@blocknote/core";
-import {
-  Button,
-  FileInput,
-  LoadingOverlay,
-  Tabs,
-  Text,
-  TextInput,
-} from "@mantine/core";
-import {
-  ChangeEvent,
-  KeyboardEvent,
-  useCallback,
-  useEffect,
-  useState,
-} from "react";
+import { LoadingOverlay, Tabs } from "@mantine/core";
+import { FC, useState } from "react";
 
 import { useBlockNoteEditor } from "../../../hooks/useBlockNoteEditor";
 import { Toolbar } from "../../mantine-shared/Toolbar/Toolbar";
 import { ImageToolbarProps } from "../ImageToolbarProps";
+import { UploadPanel } from "./DefaultTabPanels/UploadPanel";
+import { EmbedPanel } from "./DefaultTabPanels/EmbedPanel";
 
+/**
+ * By default, the ImageToolbar component will render with default tabs.
+ * However, you can override the tabs to render by passing the `tabs` prop. You
+ * can use the default tab panels in the `DefaultTabPanels` directory or make
+ * your own using the `ImageToolbarPanel` component.
+ */
```

Comment:
Is `ImageToolbar` the right name? It doesn't really feel like a toolbar, right? (as it's a lot larger)

Reply:
True, maybe `ImagePanel`?